scripts/build/download-tool.sh

```diff
@@ -78,7 +78,8 @@ if [ -f "$ARGV_OUTPUT" ]; then
   fi
 fi
 
-wget --no-check-certificate "$ARGV_URL" -O "$TEMP_OUTPUT"
+echo "Downloading $ARGV_URL"
+curl --continue-at - --retry 100 --location --output "$TEMP_OUTPUT" "$ARGV_URL"
```

Comment:
The thing is that `wget` does handle all the problems by default. In comparison, `curl` will bail out at the first minimal error. Since `wget` did all these before, it sounds reasonable that we should do it here as well.